binaries/cli/src/main.rs

```diff
@@ -232,11 +232,13 @@ enum Lang {
     Cxx,
 }
 
-fn main() {
+fn main() -> Result<()> {
+    simple_eyre::install()?;
```

Comment:
I tried to find some example how the `simple_eyre` output differs from the standard one, but I didn't find anything clear. Could you give a small example of the new formatting?

Reply:
Basically, it removes at the end:

```bash
    Location:
        /home/peter/Documents/work/dora/binaries/coordinator/src/listener.rs:90:56
```

Did not find any other way to do that

Reply:
I think we should also be able to use `{:#}` formatting instead of `{:?}` formatting: https://docs.rs/eyre/latest/eyre/struct.Report.html#display-representations

But I'm fine with `simple-eyre` too.